osbuild/meta.py

```diff
@@ -410,6 +410,42 @@ def _parse_caps(cls, _klass, _name, node):
 
     @classmethod
     def load(cls, root, klass, name) -> Optional["ModuleInfo"]:
+        base = cls.MODULES.get(klass)
+        if not base:
+            raise ValueError(f"Unsupported type: {klass}")
+        path = os.path.join(root, base, name)
+
+        try:
+            return cls._load_from_json(path, klass, name)
+        except FileNotFoundError:
+            # should we print a deprecation warning here?
+            pass
+        return cls._load_from_py(path, klass, name)
+
+    @classmethod
+    def _load_from_json(cls, path, klass, name) -> Optional["ModuleInfo"]:
+        # ideas welcome for a better filename/suffix :)
+        meta_json_suffix = "-meta.json"
+        with open(path + meta_json_suffix, encoding="utf-8") as fp:
+            meta = json.load(fp)
+
+        long_description = meta.get("description", "no description provided")
+        if isinstance(long_description, list):
+            long_description = "\n".join(long_description)
```

Comment:
I understand why, and I can't come up with a better plan, but I don't like having to join a list to have multi-line strings in JSON :(

Reply:
Yeah, it's really not great but I couldn't think of a better way (with plain json)